src/Orleans/Orleans.csproj

```diff
@@ -38,6 +38,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="$(MicrosoftExtensionsDependencyInjection)" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="1.1.2" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="1.1.2" />
```

Comment:
Console is no longer necessary here, right?

Reply:
sorry for the bits and pieces of uncleanness. I did another round of clean up..